nixos/modules/services/misc/passopolis.nix

```diff
@@ -0,0 +1,63 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.passopolis;
+in
+{
+
+  ###### interface
+
+  options = {
+
+    services.passopolis = {
+
+      enable = mkOption {
+        default = false;
+        description = "
+          Enable passopolis password manager service.
+        ";
+      };
+
+      db_url = mkOption {
+        default = "jdbc:postgresql://localhost:5432/passopolis";
```

Comment:
The only way according the the team developing it to set that db_url is as a runtime parameter (there's no app.properties or anything yet to store parameters), so if someone launches the app and has credentials in the db_url, anyone on the system can run a ps and see those credentials. Once a config system is added for the application, I completely agree that keeping the credentials out of the nix store is the better way to go or better yet encrypting in the store if RFC 0005 gets implemented.